index.html

```diff
@@ -284,12 +284,15 @@ <h2>
         If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
       </p>
       
-      <input name="name" type="text" maxlength="30" placeholder="name" required>
-  
-      <input name="email" type="email" placeholder="email" required>
-  
+      <label for="name">Name</label>
+      <input id="name" name="name" type="text" maxlength="30" placeholder="name" required>
+      
+      <label for="email">Email</label>
+      <input id="email" name="email" type="email" placeholder="email" required>
+          
+      <label for="message">Message</label>
```

Comment:
- [ ] kindly follow the figma design.. the label is not found if the figma version

Reply:
But the task requires me to add a label ?

Reply:
yes please you can do that... i know its necessary but the template you chose doesn't require it.